imsim/instcat.py

```diff
@@ -147,12 +219,15 @@ def __init__(self, file_name, wcs, xsize=4096, ysize=4096, sed_dir=None, edge_pi
                         logger.info('Using %d of %d objects so far.',nuse,ntot)
                     # Check if the object is on our image
                     tokens = line.strip().split()
-                    ra = float(tokens[2])
-                    dec = float(tokens[3])
+                    ra = float(tokens[2])*galsim.degrees
+                    dec = float(tokens[3])*galsim.degrees
                     #logger.debug('object at %s,%s',ra,dec)
-                    if not (min_ra <= ra <= max_ra and min_dec <= dec <= max_dec):
+                    if not (
+                        min_ra <= ra.wrap(ref_ra).deg <= max_ra
+                        and min_dec <= dec.deg <= max_dec
+                    ):
```

Comment:
Maybe cleaner to have get_radec_limits return Angles rather than floats.  Then we don't need this baroque combination of `.deg` applications.  Can do all <= tests with the angles themselves.

Reply:
Yep.  I think that is indeed cleaner.  

Might have been slightly cleaner still if Angle had an `__abs__` method enabling `max(np.abs([angle1, angle2]))` instead of `max(np.abs([angle1.deg, angle2.deg]))*galsim.degrees`.  Should I go add one, or is there a good reason not to have this?

Reply:
abs would be reasonable to add.  Just didn't occur to me that it would be useful.